components/collection/drop/MintButton.vue

```diff
@@ -6,7 +6,7 @@
     no-shadow
     size="large"
     :loading="loading"
-    :disabled="buttonMint.disabled"
+    :disabled="buttonMint.disabled && !isWalletConnecting"
```

Comment:
so when wallet is not connecting => button is not disabled, regardless of `buttonMint.disabled`?

i'm not sure that makes sense
wallet will not be connecting most of the time , no?